### PR TITLE
Feature/query operators

### PR DESCRIPTION
**Pull request by loudmouth**
Change `identifier` to `id` to be consistent with the API

Upgrade project to Xcode 8.3

### CODE REVIEW

File: Sources/Client.swift [hunk 1]
```diff
@@ -234,25 +234,25 @@ extension Client {
     /**
      Fetch a single Asset from Contentful
 
-     - parameter identifier: The identifier of the Asset to be fetched
+     - parameter id: The id of the Asset to be fetched
```
Comment (mariobodemann) [inline Sources/Client.swift]: Hmm, can we explain what `id` means, aka replace the 2nd `id` with `identifier` again? (Like you did with all the other refs?)

**Comment by mariobodemann** [comment]
And just another question: Since this seems to be a breaking change, how do you normally deal with those?

File: Sources/Client.swift (hunk 1 above)
Comment (loudmouth) [inline Sources/Client.swift]: yes :-) I can do that.

**Comment by mariobodemann** [comment]
And another question: Do you squash the commits? Otherwise your last commit contains the spelling mistake: `documatation` vs `documentation`.... ;)

**Pull request merged**